### ISSUE TITLE
Screen minimizes when typing past one line

### ISSUE THREAD

**Issue by Reece-Denne** [microsoft/azserialconsole#112]
Attempting to troubleshoot Azure Update manager and running the command `nano /var/log/azure/Microsoft.CPlat.Core.LinuxPatchExtension/0.core.log`

As soon as I get into a new line the characters go to the front of the existing line, not on a new line, and the screen size shrinks. Only way the screen resizes is by rebooting the VM. Very annoying.

Some screenshots below to demonstrate.

Serial Console size good:
![Serial Console Size Good](https://github.com/microsoft/azserialconsole/assets/165972485/4073f699-b563-41cb-af2f-12782dc8d587)

Serial Console size bad:
![Serial Console Size Bad](https://github.com/microsoft/azserialconsole/assets/165972485/c0658f2a-b12e-4d76-a388-b0b20328ec11)

**Comment by mbifeld** [comment]
Thank you for reporting this bug!  We will be working on revamping our Portal experience in our future roadmap and will be ensuring this is fixed during that process.  For our understanding, is the console still receiving all characters properly?  Meaning even though the characters are covering the front of the existing line, what gets executed is what you typed out?

**Comment by Reece-Denne** [comment]
Yes, the console is still receiving all characters properly. What gets executed is what I type out.

**Comment by mbifeld** [comment]
Okay, thank you for the additional details and reporting this bug!

**Comment by mbifeld** [comment]
Rolled out a fixed and closing as resolved. Please reopen / tag me if you are still experiencing any issues. Thank you!
